tasks/GPSDTask.cpp

```diff
@@ -1,4 +1,14 @@
-#include "GPSDTask.hpp"
+
+// workaround for name clash in 
+// usr/include/mach/policy.h and gpsd/3.15/include/gps.h
```

Comment:
Not really, the problem is that we include <libgpsmm.h> here which is including gpsd/3.15/include/gps.h using an already used system typedef. From my point of view the right fix would be to change the name in gps.h but this is out of my scope.